Automattic/jetpack
# Subscribe block: apply correct extra padding to the block with subscriptions count on

#### Fixes 
This PR fixes an issue where subscription blocks with extra padding and subscriptions count on, were displayed with broken padding in published content (in editor all looked just fine).

##### Editor view:
![Screenshot 2022-07-04 at 09 14 10](https://user-images.githubusercontent.com/2019970/177101961-e88b3d16-c912-439e-9cc4-224f1dd78874.png)

##### Published content:
![Screenshot 2022-07-04 at 09 15 26](https://user-images.githubusercontent.com/2019970/177101983-39753c41-02e8-41e1-9fa9-8e055dc49b4e.png)

  
#### Changes proposed in this Pull Request:

* apply container `div` to the `Save` component of subscribe block
  * that way styles applied via block settings are applied to the container `div` and are not overriding styles applied by component's classes
* apply `jetpack-subscribe-count` class directly to the `p` element of the subscriptions count's component
  * There is no need for the outer `div`
  * Now the DOM structure of the `views.php` resembles more closely the structure of the related `editor.js` component
  *  This required making a small amendment to the `view.scss` styles definition file. This ensures that the style is still properly applied to both, the subscribe block component rendered in published content and when rendered in editor 

#### Does this pull request change what data or activity we track or use?

No, it does not. 

#### Other information:


#### Testing instructions:






* Go to create a new post
* Insert `Subscribe` block
* In the `Subscribe` block's settings set the following
  * `Show subscribe count` under `Settings` to on
  * `Padding` under `Dimensions` to some positive number that will make debugging easier (usually the higher value the better), e.g. 50px.
  
![Screenshot 2022-07-04 at 09 23 48](https://user-images.githubusercontent.com/2019970/177103828-a9280194-7559-4dff-b682-a21a3baaeb7d.png)

* Save the post
* View the saved post and notice how the subscription's count element is misaligned.
